SEIR/setup.py

```diff
@@ -151,7 +151,8 @@ def seeding_draw(s, sim_id):
 
     elif (method == 'FolderDraw'):
         folder_path = s.seeding_config["folder_path"].as_str()
-        seeding = pd.read_csv(f'{folder_path}importation_{sim_id}.csv',
+        nfile = sim_id%len(os.listdir(folder_path))+1
```

Comment:
This will mess with the parameter loggging code.  Is this an important feature?  Additional importations are pretty easy to generate i thought

Reply:
If we make this change, we'll need to do it in such a way that it's obvious which importation goes with which simulation